systemtest/src/test/java/io/strimzi/systemtest/AbstractClusterIT.java

```diff
@@ -281,4 +288,35 @@ public String updateTopicPartitionsCountUsingPodCLI(String clusterName, String p
         return kubeClient.exec(podName, "/bin/bash", "-c",
                 "bin/kafka-topics.sh --zookeeper " + clusterName + "-zookeeper:2181 --alter --topic " + topic + " --partitions " + partitions).out();
     }
+
+    public Map<String, String> getImagesFromConfig(String configJson) {
+        kubeClient.getResourceAsJson("deployment", "strimzi-cluster-operator");
+        Map<String, String> images = new HashMap<>();
+        images.put(ZK_IMAGE, getImageNameFromJSON(configJson, ZK_IMAGE));
+        images.put(KAFKA_IMAGE, getImageNameFromJSON(configJson, KAFKA_IMAGE));
+        images.put(CONNECT_IMAGE, getImageNameFromJSON(configJson, CONNECT_IMAGE));
+        images.put(S2I_IMAGE, getImageNameFromJSON(configJson, S2I_IMAGE));
+        images.put(TO_IMAGE, getImageNameFromJSON(configJson, TO_IMAGE));
+        return images;
+    }
+
+    private String getImageNameFromJSON(String json, String image) {
+        return JsonPath.parse(json).read("$.spec.template.spec.containers[*].env[?(@.name =='" + image + "')].value").toString();
+    }
+
+    public String  getImageNameFromPod(String podName) {
+        String clusterOperatorJson = kubeClient.getResourceAsJson("pod", podName);
+        return JsonPath.parse(clusterOperatorJson).read("$.spec.containers[*].image").toString();
+    }
+
+    public JsonNode yamlFileToJSON(String filePath) {
```

Comment:
I think you can move this method to class `TestUtils` because it can be used in many places. Also, I have some suggestions for the name of variables. `filePath` is it a real path or this is just .yaml as `String`? If true, maybe we should name this variable otherwise?